src/main/java/com/softech/coursesvc/controller/CourseController.java

```diff
@@ -43,21 +44,27 @@ public ResponseEntity<CourseEntity> getCourseById(@PathVariable Long courseId)
         return new ResponseEntity<CourseEntity>(course, HttpStatus.OK);
     }else {
         return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
-
-
-    }}
+    }
+}
 
     @PostMapping("/v1/course")
     public CourseEntity createCourse(@RequestBody CourseEntity courseEntity){
         return courseSvc.createCourse(courseEntity);
 
     }
-    @PutMapping("/v1/course/{courseId}")
-    public void updateCourse(@PathVariable Long courseId, @RequestBody CourseEntity courseEntity){
-        courseSvc.updateCourse(courseId, courseEntity);
-    }
+        @PutMapping("/v1/course/{courseId}")
+        public void updateCourse(@PathVariable Long courseId, @RequestBody CourseEntity courseEntity) throws ValidationException {
+            if (courseId == null || courseEntity == null) {
```

Comment:
Insufficient validations...refer to design doc or lidten rrcording..we talked about validating id presrnt in body with id in path param

Reply:
Why return type is void

Reply:
Resplve conflictd...merge with master